ncm-metaconfig/src/main/metaconfig/ssh/client_attrs.tt

```diff
@@ -0,0 +1,28 @@
+[% spacelist = ['SendEnv', 'GlobalKnownHostsFile', 'IgnoreUnknown', 'UserKnownHostsFile' ] -%]
+[% commalist = ['Ciphers', 'HostbasedKeyTypes','HostKeyAlgorithms', 'KbdInteractiveDevices', 'KbdInteractiveDevices',
+    'MACs', 'PreferredAuthentications', 'CanonicalDomains', 'CanonicalizePermittedCNAMEs', 'KexAlgorithms'] -%]
```

Comment:
end multiline lists with a `,` and the `] -%]` on a new line. will make the diff easier when adding new keys.